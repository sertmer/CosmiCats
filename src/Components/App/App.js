import React from 'react';
import Header from '../Header/Header';
import LandingPage from '../LandingPage/LandingPage';
import Login from '../../Containers/Login/Login';
import ResearchContainer from '../ResearchContainer/ResearchContainer';
import './App.scss';
import { Route } from 'react-router-dom';

export const App = () => {
  return (
    <main>
      <Route path='/' render={ () => 
          <Header />
        }
      />
      <Route exact path='/' render={() =>
          <LandingPage />
        }
      />
      <Route path='/login' render={() =>
          <Login />
        }
      />
      <Route exact path='/research' render={() => 
          <ResearchContainer />
        }
      />
    </main>
  );
}

export default App;