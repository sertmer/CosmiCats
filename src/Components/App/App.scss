@import '../../_scss/mixins';
@import '../../_scss/variables';

main {
  background-color: $dark-blue;
  height: 100vh;
  display: flex;
  flex-direction: column;
  padding: 1%;
}