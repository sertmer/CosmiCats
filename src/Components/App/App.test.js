import React from 'react';
import { shallow } from 'enzyme';
import { App } from './App';

describe('App', () => {

  let wrapper

  beforeEach(() => {
    wrapper = shallow(<App />)
  })

  describe('App unit tests', () => {
    it.skip('should match the snapshot', () => {
      expect(wrapper).toMatchSnapshot()
    })
  })
})