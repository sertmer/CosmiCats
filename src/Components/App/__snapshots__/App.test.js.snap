// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App unit tests should match the snapshot 1`] = `
<main>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Header)",
        "type": [Function],
      }
    }
    path="/"
  />
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(LandingPage)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Login)",
          "type": [Function],
        }
      }
      path="/login"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(ResearchContainer)",
          "type": [Function],
        }
      }
      exact={true}
      path="/research"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(EvidenceContainer)",
          "type": [Function],
        }
      }
      exact={true}
      path="/evidence"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</main>
`;
